crypto/src/note.rs

```diff
@@ -403,11 +403,11 @@ impl From<&Note> for [u8; NOTE_LEN_BYTES] {
     fn from(note: &Note) -> [u8; NOTE_LEN_BYTES] {
         let mut bytes = [0u8; NOTE_LEN_BYTES];
         bytes[0] = NOTE_TYPE;
-        bytes[1..12].copy_from_slice(&note.diversifier.0);
-        bytes[12..20].copy_from_slice(&note.value.amount.to_le_bytes());
-        bytes[20..52].copy_from_slice(&note.value.asset_id.0.to_bytes());
-        bytes[52..84].copy_from_slice(&note.note_blinding.to_bytes());
-        bytes[84..116].copy_from_slice(&note.transmission_key.0);
+        bytes[1..17].copy_from_slice(&note.diversifier.0);
+        bytes[17..25].copy_from_slice(&note.value.amount.to_le_bytes());
+        bytes[25..57].copy_from_slice(&note.value.asset_id.0.to_bytes());
+        bytes[57..89].copy_from_slice(&note.note_blinding.to_bytes());
+        bytes[89..121].copy_from_slice(&note.transmission_key.0);
```

Comment:
While we're at it, we should remove the `NOTE_TYPE` byte. We don't use it currently, and if we want to version note formats, we should be doing that at the level of the protos, not inside the ciphertexts.